docs/criusupport.md

```diff
@@ -0,0 +1,99 @@
+
+| Options                    | Changed behavior  |
+|----------------------------|---------|
+| [`-Xverbosegclog`](xverbosegclog.md) |      |
+| [`-XcompilationThreads`](xcompilationthreads.md) |     |
+| [`-XsamplingExpirationTime`](xsamplingexpirationtime.md)     |     |
+| [`-XX:[+|-]PrintCodeCache`](xxprintcodecache.md)        |     |
+| [`-Xtrace`](xtrace.md) | If you specify an `output` file before a checkpoint and another `output` file for restore, both the files are written according to the `-Xtrace` options associated with the `output` file. |
+| [`-Xdump`](xdump.md) | Dump events that are triggered on exception throws or catches cannot be enabled on restore. |
+| [`-Xgcthreads`](xgcthreads.md) | This option is ignored if the number of GC threads is less than the checkpoint GC thread count. |
+| [`-Xjit`](xjit.md) | If `-Xint` or `-Xnoaot` and `-Xnojit` are specified pre-checkpoint, the compiler remains disabled post-restore. |
+| [`-Xaot`](xaot.md) | If `-Xnoaot` is specified pre-checkpoint, then specifying `-Xaot` post-restore does not enable AOT compilation or load. |
+| [`-Xjit`](xjit.md) / [`-Xaot`](xaot.md)  | You can specify the following parameters with the `-Xjit` and `-Xaot` options when you run a restored VM:<br>`count`: Applies only to new methods; existing interpreted methods will not have their counts that are updated.<br> `limit`, `limitFile`, `exclude`: Invalidates the existing compiled methods that do not match the filters. The filters are then enforced for all future JIT and AOT compilations.<br> `loadLimit`, `loadLimitFile`, `loadExclude`: Applies only to future AOT loads; does not impact existing compiled methods.<br> `verbose`: Effective post-restore; does not override verbose options specified pre-checkpoint.<br> `vlog`: A vlog file is opened post-restore. If this option was specified pre-checkpoint, the old file is closed first.    |
+| [`-Xnoaot`](xaot.md) | Prevents AOT compilations and loads; does not affect the existing compiled methods and does not prevent JIT compilation.  |
+| [`-Xnojit`](xjit.md#syntax) | Invalidates all existing compiled methods and prevents JIT compilations; does not prevent AOT compilations and loads.  |
+
+If you specify an unsupported option, the VM throws a `JVMRestoreException` error by default. If you specify the `-XX:+IgnoreUnrecognizedRestoreOptions` option, the VM does not throw any exception and just ignores the unsupported option.
+
+For more more information on CRIU support, see the `openj9.criu` module in your OpenJ9 JDK version (OpenJDK 11 and later) API documentation.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:**
+
+- If an `-Xtrace` or JIT log output file is specified on startup and doesn’t exist on restore, or is modified in any way between checkpoint and restore, the restore operation fails. This limitation will be removed in the future.
+- Method tracing might not work consistently if a method is compiled pre-checkpoint.
```

Comment:
"This limitation will be removed in the future" : is there a particular reason this sort of future guidance is provided in this place but not in a host of other places ?

Reply:
I have removed this to make it consistent across. Thanks!